Sources/Sentry/SentryInstallation.m

```diff
@@ -1,13 +1,14 @@
 #import "SentryInstallation.h"
 #import "SentryDefines.h"
+#import "SentryOptions.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
 @implementation SentryInstallation
 
 static NSString *volatile installationString;
 
-+ (NSString *)id
++ (NSString *)idWithOptions:(SentryOptions *)options
```

Comment:
`m`: As we only need the cacheDirectory, we can change this to.
```suggestion
+ (NSString *)idWithOptions:(NSString *)cacheDirectory
```

Reply:
I think you would then change the method/parameter name too. And would this also have a default if supplying `nil`?
```suggestion
+ (NSString *)idWithCacheDirectory:(nullable NSString *)cacheDirectory
```